health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/PlanInbox.js

```
@@ -413,23 +491,28 @@ const PlanInbox = () => {
       const reorderedStatusCount = Object.fromEntries(
         Object.entries(planWithCensus?.StatusCount || {})
-          // Filter out the PENDING_FOR_APPROVAL status /// need to revisit as this is hardcoded to remove from workflow ///
+          // Filter out PENDING_FOR_APPROVAL (if needed, can be removed later)
           .filter(([key]) => key !== "PENDING_FOR_APPROVAL")
-          // Sort the statuses, prioritizing PENDING_FOR_VALIDATION
+          // Sort based on the order defined in filterConfig
           .sort(([keyA], [keyB]) => {
-            if (keyA === "PENDING_FOR_VALIDATION") return -1;
-            if (keyB === "PENDING_FOR_VALIDATION") return 1;
-            return 0;
+            return (statusOrderMap[keyA] || Infinity) - (statusOrderMap[keyB] || Infinity);
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)